### ISSUE TITLE
Adaptive ylim of panels

### ISSUE THREAD

**Issue by rossb34** [joshuaulrich/xts#139]
The series is not plotted correctly if its `ylim` is outside of the `ylim` of the panel it is being drawn on.

```
library(xts)
set.seed(1234)
y1 <- .xts(rnorm(50, 0, 1), 1:50)
y2 <- .xts(rnorm(50, 0, 3), 1:50)

```

Note that `y2` has a larger range than `y1`

```
> range(y1)
[1] -2.345698  2.415835
> range(y2)
[1] -5.418094  7.646973
```

Several data points of `y2` are not visible when I plot `y1` and then add `y2` with `lines`.

```
plot(y1, col="red", lwd=1)
lines(y2, col="blue")
```

![rplot](https://cloud.githubusercontent.com/assets/5797430/15595089/9921469a-2381-11e6-9068-9aa3cfd43a73.png)
Here is what the plot should look like with its `ylim` based on `y2`
![rplot03](https://cloud.githubusercontent.com/assets/5797430/15595108/d793e432-2381-11e6-8a77-83e3025e7a65.png)

**Comment by rossb34** [comment]
Thanks to some "jeff magic" code there is a way to specify if the axis should be fixed when adding a frame and setting the ylim. We should be able to support adaptive ylim by setting `fixed=false`. The trick is to use the `ylim`variable in the `Env` environment of the chart object.

**Comment by joshuaulrich** [comment]
Closing because this is no longer an issue. It was addressed as part of #360.
